controller.go

```diff
@@ -347,6 +347,7 @@ func (c *controller) NewNetwork(networkType, name string, options ...NetworkOpti
 		ctrlr:       c,
 		persist:     true,
 		drvOnce:     &sync.Once{},
+		isolated:    true,
```

Comment:
Default behavior is that the network has or will be provided outside connectivity.
This line can be removed